app/src/main/java/org/mifos/mobile/ui/fragments/HelpFragment.kt

```diff
@@ -121,12 +129,25 @@ import javax.inject.Inject
             }
 
             override fun onQueryTextChange(newText: String): Boolean {
-                faqAdapter?.updateList(presenter?.filterList(faqArrayList, newText))
+                val filteredList = presenter?.filterList(faqArrayList, newText)
```

Comment:
Let's use a better choice of variable names like `filteredFAQList` or something better.

Reply:
Oh thanks, I will the PR with the required changes